dev/benchmarks/test_apps/stocks/lib/stock_data.dart

```diff
@@ -19,7 +19,7 @@ class Stock {
   Stock(this.symbol, this.name, this.lastSale, this.marketCap, this.percentChange);
 
   Stock.fromFields(List<String> fields) {
-    // FIXME: This class should only have static data, not lastSale, etc.
+    // TODO(jackson): This class should only have static data, not lastSale, etc.
```

Comment:
Does this really require a TODO?  This code is really only used for benchmarking at this point.  Maybe just leave the comment about how it would be a better design, but it's being left this way for simplicity?

/cc @collinjackson

Reply:
Sure thing if everyone is on board. :) There are a lot of TODOs like that.